prow/manifests/overlays/metal3/kustomization.yaml

```diff
@@ -66,3 +66,30 @@ secretGenerator:
   name: cherrypick-bot-github-token
   namespace: prow
   type: Opaque
+
+images:
+- name: gcr.io/k8s-prow/crier
+  newTag: v20230329-c93d79fb7d
+- name: gcr.io/k8s-prow/deck
+  newTag: v20230329-c93d79fb7d
+- name: gcr.io/k8s-prow/ghproxy
+  newTag: v20230329-c93d79fb7d
+- name: gcr.io/k8s-prow/hook
+  newTag: v20230329-c93d79fb7d
+- name: gcr.io/k8s-prow/horologium
+  newTag: v20230329-c93d79fb7d
+- name: gcr.io/k8s-prow/prow-controller-manager
+  newTag: v20230329-c93d79fb7d
+- name: gcr.io/k8s-prow/sinker
+  newTag: v20230329-c93d79fb7d
```

Comment:
is this newTag a keyword which is identified by kustomize / go code ?

Reply:
Yes by kustomize